### ISSUE TITLE
Ore compat

### ISSUE THREAD

**Issue by codetaylor** [codetaylor/pyrotech#59]
The ore compat system has changed since its initial conception.

It is currently set up to work with the following oreDict / slagColor combinations:

```java
this.put("oreAluminum", "e2e2e3");
this.put("oreCopper", "ffaa1c");
this.put("oreGold", "fcee4b");
this.put("oreIridium", "d5d4e7");
this.put("oreIron", "d8af93");
this.put("oreLead", "95afee");
this.put("oreMithril", "9cf9ff");
this.put("oreNickel", "e4e5d8");
this.put("orePlatinum", "55f3ff");
this.put("oreSilver", "e8f6fd");
this.put("oreTin", "c3e9ff");
```

The auto-detection system works if there is at least one corresponding nugget oredicted(?) to the same thing, ie. `oreIron` has at least one entry and `nuggetIron` has at least one entry.

### If you can think of an oreDict / hexColor to add to this list, please post a comment.

**Comment by nihiltres** [comment]
Some additions, based on what I'm using in an experimental instance, formatted to match your example:
```java
this.put("oreUranium", "5c734b");
this.put("oreSyrmorite", "4455bb");
this.put("oreOctine", "ffaa22");
this.put("oreCobalt", "1c5bc1");
this.put("oreArdite", "e85e17");
```

Uranium is Immersive Engineering, Syrmorite and Octine is the Betweenlands, and Cobalt and Ardite are from Tinker's Construct.

**Comment by NightmareTwilight** [comment]
oreOsmium from mekanism (and any other that add/use it), though I'm not good with hexcolor so I'm not sure what it should be.

**Comment by nihiltres** [comment]
I'd suggest
```java
this.put("oreOsmium", "9babc4");
```
by sampling from the ingot image.

**Comment by codetaylor** [comment]
@nihiltres Thank you! The data you have provided has been added and will be available in 1.1.0
